src/dreamchecker/lib.rs

```diff
@@ -1873,6 +1874,23 @@ impl<'o, 's> AnalyzeProc<'o, 's> {
         }
     }
 
+    fn check_type_sleepers(&mut self, ty: TypeRef<'o>, location: Location, unscoped_name: &Ident) {
+        println!("{}, {}", ty, unscoped_name);
```

Comment:
Garbage on stdout will kill the langserver. You can use `eprintln!()` to avoid this problem (or delete this line since it's just debugging junk)

Reply:
ooops